registrations/models.py

```diff
@@ -73,6 +78,26 @@ class Registration(models.Model):
     def __str__(self):
         return str(self.id)
 
+    def get_voice_days_and_times(self):
+        return self.data.get('voice_days'), self.data.get('voice_times')
+
+    def get_preg_week(self):
```

Comment:
Considering this is on the `Registration` model calling it `get_week` could lead to some confusion as to it returning the week of what, the registration or the pregnancy?

I'm happy to rename it but I think `get_week()` adds to much ambiguity and isn't an improvement.